ecssd_agent.go

```diff
@@ -21,8 +21,7 @@ import (
 )
 
 const workerTimeout = 180 * time.Second
-const defaultTTL = 0
-const defaultWeight = 1
+const defaultTTL = 60
```

Comment:
Document that this TTL is in seconds.

https://godoc.org/github.com/aws/aws-sdk-go/service/route53#ResourceRecordSet

>  // The resource record cache time to live (TTL), in seconds.